FINE/utils.py

```diff
@@ -462,6 +466,66 @@ def checkAndSetCostParameter(esM, name, data, dimension, locationEligibility):
     return _data
 
 
+def checkAndSetTimeSeriesCostParameter(esM, name, data, locationEligibility, dimension = 1):
```

Comment:
Could we change locationEligibility to locationalEligibility here? This would be more consistent with the rest of the code.